lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -5,6 +5,22 @@
     </a>
 <% }else{ %>
     <div class="enrollment-info"><%- gettext('not enrolled') %></div>
+    <% if (run_modes.length > 1){ %>
+        <div class="run-select-container">
+            <label class="sr-only">Select Course Run</label>
```

Comment:
Label must either wrap the form control (implicit association) or contain a ```for``` attribute that references the associated form control by IDref: 
https://www.w3.org/TR/html5/forms.html#the-label-element